### PR TITLE
GH-Actions: adding security actions and scorecard

### PR DESCRIPTION
**Pull request by lucbv**
@csiefer2 let me know if you have comments on this as well?
Pretty much taking the new files from PR #2191 and re-creating it in a clean PR on top of develop with small changes relevant to our repository.

### CODE REVIEW

**Comment by lucbv** [comment]
Adding WIP as we do not really want to run that through the auto-tester...

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: ["master", "develop", "release-*"]
+  pull_request:
```
[the remark this answers is not in the document]
Comment (lucbv) [reply .github/workflows/codeql.yml:17]: Yeah, in our case that could potentially avoid issues when we are doing releases which include some cherry picking and could led to problems. That's something that we can turn off later easily if it really creates any undue burden.

File: .github/.pre-commit-config.yaml
[the remark this answers is not in the document]
Comment (lucbv) [reply .github/.pre-commit-config.yaml:1]: That's quite true, I'll axe this one...

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CUDA11_CUDA11_LayoutRight</h3>
<ul>
  <li>Build Num: 1297</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930_Tpls_CLANG13CUDA10</h3>
<ul>
  <li>Build Num: 886</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GNU1021</h3>
<ul>
  <li>Build Num: 543</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GNU1021_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 530</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GNU1021</h3>
<ul>
  <li>Build Num: 531</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL19_solo</h3>
<ul>
  <li>Build Num: 535</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001_solo</h3>
<ul>
  <li>Build Num: 507</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_Tpls_ARMPL2110</h3>
<ul>
  <li>Build Num: 1017</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_GCC1020</h3>
<ul>
  <li>Build Num: 1012</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_VEGA90A_ROCM561</h3>
<ul>
  <li>Build Num: 1003</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_VEGA90A_Tpls_ROCM561</h3>
<ul>
  <li>Build Num: 525</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (lucbv/kokkos-kernels)</dt>
  <ul>
    <li>Branch: security</li>
    <li>SHA: f2126afdc8bca8e9390be0a3411408afe66837d8</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: lucbv
</details>

**Review by brian-kelley — APPROVED** [review]

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CUDA11_CUDA11_LayoutRight</h3>
<ul>
  <li>Build Num: 1297</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930_Tpls_CLANG13CUDA10</h3>
<ul>
  <li>Build Num: 886</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GNU1021</h3>
<ul>
  <li>Build Num: 543</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GNU1021_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 530</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GNU1021</h3>
<ul>
  <li>Build Num: 531</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL19_solo</h3>
<ul>
  <li>Build Num: 535</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001_solo</h3>
<ul>
  <li>Build Num: 507</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_Tpls_ARMPL2110</h3>
<ul>
  <li>Build Num: 1017</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_GCC1020</h3>
<ul>
  <li>Build Num: 1012</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_VEGA90A_ROCM561</h3>
<ul>
  <li>Build Num: 1003</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_VEGA90A_Tpls_ROCM561</h3>
<ul>
  <li>Build Num: 525</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/lucbv/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f2126afdc8bca8e9390be0a3411408afe66837d8</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>88ae8f1489e390578485c9c274a2129d38da5301</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td></td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2192</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ brian-kelley ]!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - This Repo does not support Automerge

**Pull request merged**